auth_provider/templates/auth/login.html

```diff
@@ -1,23 +1,108 @@
-{% extends "base.html" %} {% block content %} {% if form.errors %}
-<div class="alert">
-  <p>Username and password did not match</p>
-</div>
-{% endif %}
+{% extends "base.html" %} {% block content %} 
+
+<style>
+  body {
+    font-family: Arial, Helvetica, sans-serif;
+  }
+
+  * {
+    box-sizing: border-box;
+  }
+
+  /* Add padding to containers */
+  .container {
+    padding-top: 20px;
+    padding-right: 50px;
+    padding-bottom: 20px;
+    padding-left: 50px;
+    background-color: white;
+  }
```

Comment:
These seem reusable let's put them under `styles.css`

Reply:
Sure, that's doable!